conda/recipes/raft-dask/conda_build_config.yaml

```diff
@@ -14,7 +14,7 @@ sysroot_version:
   - "2.17"
 
 ucx_version:
-  - ">=1.13.0,<1.15.0"
+  - ">=1.13.0,<1.16.0"
```

Comment:
```suggestion
  - ">=1.14.1,<1.16.0"
```

Reply:
Just to make it [consistent with UCX-Py](https://github.com/rapidsai/ucx-py/blob/c34aad0e163a5f656a58d57dcf03530a948ec700/conda/recipes/ucx-py/meta.yaml#L38).